tools.go

```diff
@@ -0,0 +1,11 @@
+//go:build tools
+// +build tools
```

Comment:
I don't think you need both comments, the first one `go:build` should be enough. `+build` is just for compatibility with pre-`go 1.17` versions. For details see: https://golang.org/doc/go1.18#go-build-lines